cmd/minikube/cmd/service.go

```diff
@@ -154,10 +157,22 @@ You may select another namespace by using 'minikube service {{.service}} -n <nam
 			}
 		}
 
+		noNodePortSvcNames := []string{}
+		for _, svc := range noNodePortServices {
+			noNodePortSvcNames = append(noNodePortSvcNames, fmt.Sprintf("%s/%s", svc.Namespace, svc.Name))
+		}
+		if len(noNodePortServices) != 0 {
+			out.WarningT("Services {{.svc_names}} have type \"ClusterIP\" . Minikube allows you to access them only for testing", out.V{"svc_names": noNodePortSvcNames})
```

Comment:
minikube should be lowercase

Reply:
Services {{.svc_names}} have type \"ClusterIP\" not meant to be exposed, however for local developement minikube allows you to access this !"